.travis.yml

```diff
@@ -16,7 +16,7 @@ before_script:
   - rustup component add clippy
 script:
 - |
-  DCO_SIGNING_BASE_COMMIT=4c9e80eb9ddb5f97e2e55c63a8cfb258897ceac5 &&
+  DCO_SIGNING_BASE_COMMIT=b00f12dc7afa4400af1289a7f69e570148ae16e5 &&
```

Comment:
This was the previous master commit. We were otherwise fine since July 2019. What happened that broke this?

Reply:
This was not the previous master commit. 4c9e80eb9ddb is 5.2.1, master is at 6.0.1.

I don't know how we got from 5.2.1 to 6.0.1 without hitting this.

Reply:
By "this" I meant the one you changed it to, https://github.com/slog-rs/extlog/commit/b00f12dc7afa4400af1289a7f69e570148ae16e5

Reply:
Then I think we agree and are both mystified.